subsys/spm/Kconfig

```diff
@@ -83,6 +83,16 @@ config SPM_SERVICE_FIND_FIRMWARE_INFO
 	  marked as secure. This service allows it to request firmware info
 	  about image stored at a given address.
 
+config SPM_SERVICE_PREVALIDATE
+	bool "Prevalidate B1 upgrades"
+	default y
+	select SECURE_BOOT_CLIENT_API
```

Comment:
I thin this should be `depends on`

Ref: https://docs.zephyrproject.org/latest/guides/kconfig/index.html#select-recommendations
 "Avoid selecting symbols with prompts or dependencies. Prefer depends on"